portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,25 +31,105 @@
 
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> quotes;
-
-  @Override
-  public void init() {
-    quotes = new ArrayList<>(4);
-    quotes.add("Hope you are having a good one!");
-    quotes.add("How are you :)");
-    quotes.add("It is fantastic to see you!");
-    quotes.add("Thanks for vising my portfolio!");
-  }
+  private UserComment comment;
+  private static final int DEFAULT_MAX_COMMENTS = 10;
```

Comment:
Is it possible to display this default in the "max number" text box?

Reply:
Yes, I'll add this

Reply:
This is done.